### PR TITLE
Add owner to query users request

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candidclient/3/

File: params/params.go [hunk 1]
```diff
@@ -106,6 +106,10 @@ type QueryUsersRequest struct {
 	// if using Time.MarshalText. It matches all identies that have a
 	// last discharge time after the given time.
 	LastDischargeSince string `httprequest:"last-discharge-since,form"`
+
+	// Owner, if present, matches all agent identities with the given
```
Comment (rogpeppe) [inline params/params.go:110]: We're planning to make this not just agent identities, right?

**Review by rogpeppe — APPROVED** [review]
LGTM with one minor thought, thanks.

**Review by hatched — APPROVED** [review]

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candidclient-merge

**Pull request merged**